src/admin/resolvers/queries/ApprovedItem.auth.integration.ts

```diff
@@ -0,0 +1,264 @@
+import { expect } from 'chai';
+import { CuratedStatus } from '@prisma/client';
+import { ACCESS_DENIED_ERROR, MozillaAccessGroup } from '../../../shared/types';
+import { db } from '../../../test/admin-server';
+import {
+  clearDb,
+  createApprovedItemHelper,
+  getServerWithMockedHeaders,
+} from '../../../test/helpers';
+import {
+  GET_APPROVED_ITEMS,
+  GET_APPROVED_ITEM_BY_URL,
+} from './sample-queries.gql';
+
+describe('queries: ApprovedCuratedCorpusItem - authentication', () => {
+  beforeAll(async () => {
+    // clear out db to start fresh
+    await clearDb(db);
+  });
+
+  afterAll(async () => {
+    await db.$disconnect();
+  });
+
+  describe('getApprovedCuratedCorpusItems query', () => {
+    beforeAll(async () => {
+      // Create some items
+      const stories = [
+        {
+          title: 'How To Win Friends And Influence People with GraphQL',
+          language: 'en',
+          status: CuratedStatus.RECOMMENDATION,
+          topic: 'FOOD',
+        },
+        {
+          title: 'What Zombies Can Teach You About GraphQL',
+          language: 'en',
+          status: CuratedStatus.RECOMMENDATION,
+          url: 'https://www.sample-domain/what-zombies-can-teach-you-graphql',
+          topic: 'TECHNOLOGY',
+        },
+        {
+          title: 'How To Make Your Product Stand Out With GraphQL',
+          language: 'en',
+          status: CuratedStatus.RECOMMENDATION,
+          topic: 'TECHNOLOGY',
+        },
+      ];
+
+      // insert test stories into the db
+      for (const story of stories) {
+        await createApprovedItemHelper(db, story);
+      }
+    });
+
+    afterAll(async () => {
+      await db.$disconnect();
+    });
```

Comment:
I don't think this second DB disconnect should be in this file at all - the one within the top-level `describe()` block is probably enough.